src/slack/blocks/postAnonymousReply.ts

```diff
@@ -0,0 +1,26 @@
+import { KnownBlock, InputBlock } from '@slack/types';
+
+export function getPostAnonymousReplyBlock(originalPost: string): KnownBlock[] {
```

Comment:
This method name says "block" but it's returning an array. Maybe this should be `getPostAnonymousReplyViewBlocks`?